### PR TITLE
update periodic isolated cleanup to match the new paths post-runner

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

**Comment by ryanpetrello** [comment]
@AlanCoding these paths changed in the move to runner:

```shell
[root@ip-10-0-10-224 ec2-user]# ansible-runner --version && ls -la /tmp
1.3.1
total 4
drwxrwxrwt. 13 root root 4096 Apr  4 21:06 .
dr-xr-xr-x. 18 root root  236 Oct 17 12:25 ..
drwx------.  5 awx  awx    57 Apr  4 21:01 ansible_runner_pi_IdYlrH
drwx------.  5 awx  awx    57 Apr  4 20:32 ansible_runner_pi_NHQMvy
drwx------.  5 awx  awx    57 Apr  4 20:44 ansible_runner_pi_ZFNCuW
drwx------.  7 awx  awx   147 Apr  4 20:44 awx_136_y_7mohmt
drwx------.  7 awx  awx   147 Apr  4 21:01 awx_138_h_nfbq9s
```

**Comment by ryanpetrello** [comment]
recheck

**Comment by ryanpetrello** [comment]
recheck

File: awx/plugins/isolated/awx_isolated_cleanup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def main():
     job_cutoff = datetime.datetime.now() - datetime.timedelta(hours=1)
 
     for search_pattern in [
-        '/tmp/ansible_awx_[0-9]*_*', '/tmp/ansible_awx_proot_*',
+        '/tmp/awx_[0-9]*_*', '/tmp/ansible_runner_pi_*',
```
Comment (AlanCoding) [inline awx/plugins/isolated/awx_isolated_cleanup.py:42]: checks out

https://github.com/ansible/ansible-runner/blob/c70a02670882d94ae189a32b77a5f07df205dab9/ansible_runner/runner_config.py#L370

File: awx/plugins/isolated/awx_isolated_cleanup.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def main():
                 continue
             elif modtime > folder_cutoff:
                 try:
-                    re_match = re.match(r'\/tmp\/ansible_awx_\d+_.+', path)
+                    re_match = re.match(r'\/tmp\/awx_\d+_.+', path)
```
Comment (AlanCoding) [inline awx/plugins/isolated/awx_isolated_cleanup.py:52]: https://github.com/ansible/awx/blob/9bc5d915e12034169088642c29cf4adfa83e2e4c/awx/main/isolated/manager.py#L343

update comment?

Comment (AlanCoding) [reply awx/plugins/isolated/awx_isolated_cleanup.py:52]: also, I believe this is a relevant bit here:

https://github.com/ansible/awx/blob/9bc5d915e12034169088642c29cf4adfa83e2e4c/awx/main/tasks.py#L759

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**